MinStack.java

```diff
@@ -0,0 +1,62 @@
+// Time Complexity :O(1) in each operation
+// Space Complexity :O(2n) worst case
+// Did this code successfully run on Leetcode : Yes
+// Any problem you faced while coding this :
+//  - Stack stack = new Stack(); why is this not functioning at line 14
```

Comment:
Please describe any difficulties you had in comprehending the logic or making the code work and not compiler error output messages